src/tests/ftest/util/daos_core_base.py

```diff
@@ -106,8 +106,10 @@ def start_server_managers(self):
     def run_subtest(self):
         """Run daos_test with a subtest argument."""
         subtest = self.params.get("daos_test", self.TEST_PATH)
-        num_clients = self.params.get("num_clients",
-                                      '/run/daos_tests/num_clients/*')
+        num_clients = self.params.get("num_clients", self.TEST_PATH)
```

Comment:
@phender im looking for your feedback if this is OK (and the other change to the dfs.py file to leverage the DaosCoreBase.